numpy/core/include/numpy/npy_math.h

```diff
@@ -77,6 +77,7 @@ NPY_INLINE static float __npy_nzerof(void)
 #define NPY_LOGE2     0.693147180559945309417232121458176568  /* log_e 2 */
 #define NPY_LOGE10    2.302585092994045684017991454684364208  /* log_e 10 */
 #define NPY_PI        3.141592653589793238462643383279502884  /* pi */
+#define NPY_TAU       6.283185307179586476925286766559005768  /* tau = 2*pi */
```

Comment:
Might as well add the float and long double versions as well.

Reply:
We don't appear to use `2*pi` internally, but it does seem like something we might want some day so it doesn't hurt to define it for all types.

Reply:
Didn't see those.

On that note, what's the correct way to get `np.longdouble(np.pi)`? Obviously that expression isn't as precise as it could be.